public/locales/en.json

```diff
@@ -133,7 +133,7 @@
         }
     },
     "editor": {
-        "placeholder": "← Start by entering a title here\n===\nVisit /features if you don't know what to do.\nHappy hacking :)",
+        "placeholder": "← Start by entering a title here\n===\nVisit /n/features if you don't know what to do.\nHappy hacking :)",
```

Comment:
is that necessary since /features redirects to /n/features?

Reply:
Is the change mentioned by @SISheogorath okay for you?

You're right with the redirect, you could apply that comment to the whole PR. I just thought why should our links in the application have the overhead of a redirect, when they could point to the right target directly. So I did a quick regex replace.

Personally I like "the features page" even more than just "/features", but if you don't like that, I can change that back.

Reply:
I think this PR is a good idea and I for one like 'the features page' also more than `/n/features`